### PR TITLE
Update minimessage format example of click command

### PR DESCRIPTION
**Pull request by lynxplay**
Since chat signatures were introduced in 1.19.1, the client no longer executes commands defined in a components click action if they require a signed argument, such as the /say command.

This commit updates the example to something that works as expected, the /seed command.

### CODE REVIEW

**Comment by kezz** [comment]
We should probably have a little callout box here that mentions not being able to run signed commands. Similar to what we have on the sounds page for mono/stereo sounds.

**Comment by lynxplay** [comment]
I'll add it :+1:

**Review by MiniDigger — APPROVED** [review]

**Review by kezz — APPROVED** [review]
LGTM! Thanks for the contribution :)

File: source/minimessage/format.rst [hunk 1]
```diff
@@ -138,11 +138,15 @@ Arguments
    * ``_action_``, the type of click event, one of `this list <https://jd.advntr.dev/api/latest/net/kyori/adventure/text/event/ClickEvent.Action.html#enum.constant.summary>`_
    * ``_value_``, the argument for that particular event, refer to `the minecraft wiki <https://minecraft.gamepedia.com/Raw_JSON_text_format>`_
 Examples
-   * :mm:`<click:run_command:/say hello>Click</click> to say hello`
+   * :mm:`<click:run_command:/seed>Click</click> to show the world seed!`
    * :mm:`Click <click:copy_to_clipboard:Haha you suck> this </click>to copy your score!`
```
Comment (Joo200) [inline source/minimessage/format.rst:142]: Maybe change the "Haha you suck" too...

File: source/minimessage/format.rst [hunk 2]
```diff
@@ -138,11 +138,15 @@ Arguments
...
 
 .. image:: /minimessage/images/click_1.png
-   :alt: The result of parsing ``<click:run_command:/say hello>Click</click> to say hello``, shown in-game in the Minecraft client's chat window
+   :alt: The result of parsing ``<click:run_command:/seedo>Click</click> to show the world seed!``, shown in-game in the Minecraft client's chat window
```
Comment (Joo200) [inline source/minimessage/format.rst]: `/seedo`?

**Review by Joo200 — COMMENTED** [review]
Found 2 minor things, but other than that LGTM

File: source/minimessage/format.rst (hunk 2 above)
Comment (lynxplay) [reply source/minimessage/format.rst]: Fixed :+1:

File: source/minimessage/format.rst (hunk 1 above)
Comment (lynxplay) [reply source/minimessage/format.rst:142]: I feel like this properly represents my average gamescore, I don't mind a little easter egg, I'd leave it in unless there is actual pushback against this :sweat_smile:

**Pull request merged**